kaz.html

```diff
@@ -22,7 +24,12 @@ <h1>Apertium Paradigm Generator</h1>
 <input id='ParadigmText' placeholder="Text to see Paradigms"></input>
 <button type="button" onclick="paradigm()">See Paradigms</button><br />
 
-<input type="hidden" data-remove-tags="ger.nom"></input>
+<input type="hidden" id="noremove" data-no-remove-tags="tv.iv" />
```

Comment:
Hmmm, how does it know that `<v><iv>`, `<v><tv>`, and `<vaux>` are the patterns that it should be testing against?

Reply:
It doesn't include the subcategory when testing and there is an array of valid verb forms. This is similar to how it previously worked

Reply:
But then when it send the request to the APy, it uses the subcategory tags

Reply:
Then how does it know to get verb forms, as opposed to nouns or something?

Reply:
Currently, you can only make verb paradigms.The support for other word categories was not added in this PR, but it shouldn't be too hard to add

Reply:
Currently, you can only make verb paradigms.The support for other word categories was not added in this PR, but it shouldn't be too hard to add